crates/core/src/config/wow.rs

```diff
@@ -84,6 +95,7 @@ impl Flavor {
             Flavor::Classic => "_classic_".to_owned(),
             Flavor::ClassicPtr => "_classic_ptr_".to_owned(),
             Flavor::ClassicBeta => "_classic_beta_".to_owned(),
+            _ => panic!("Unsupported flavor: {:?}", self),
```

Comment:
Will these cases ever be hit in practice?

Reply:
It is used to convert a `Flavor` into the folder name, and those places we use this is all tied to `Flavor::ALL` which we control: 

```rust
    pub const ALL: [Flavor; 6] = [
        Flavor::Retail,
        Flavor::RetailPtr,
        Flavor::RetailBeta,
        Flavor::Classic,
        Flavor::ClassicPtr,
        Flavor::ClassicBeta,
    ];
```

I am not sure if its better to return a string, rather than just panic.